ods_ci/tests/Tests/0500__ide/0501__ide_jupyterhub/minimal-cuda-test.robot

```diff
@@ -90,7 +90,7 @@ Verify CUDA Image Suite Setup
     # This will fail in case there are two nodes with the same number of GPUs
     # Since the overall available number won't change even after 1 GPU is assigned
     # However I can't think of a better way to execute this check, under the assumption that
-    # the Resources-GPU tag will always ensure there is 1 node with 1 GPU on the cluster.
+    # the Resources-GPU    NVIDIA-GPUs tag will always ensure there is 1 node with 1 GPU on the cluster.
```

Comment:
this line change doesn't make much sense probably

Reply:
yeah sorry, i did a replace all in the repo. I can remove it, one sec

Reply:
done

Reply:
thx